src/components/icons/DefaultToken.tsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+
+interface DefaultTokenProps {
+  size?: number;
+  symbol?: string;
+}
+
+export default function DefaultToken({
+  size,
+  symbol,
+}: DefaultTokenProps): JSX.Element {
+  // const TextElement = Text as any;
+
+  const bg = "white";
```

Comment:
i see these only used in a single place, do we need to have them as a variable?

Reply:
updated to remove variable and use in fill directly